src/commands/suggest.ts

```diff
@@ -0,0 +1,47 @@
+// Codey suggest Command
+import Discord from 'discord.js';
+import { openDB } from '../components/db';
+
+/*
+  // Keeping this here just to keep my debugging tools
+  db.run('DROP TABLE suggestions');
+  const res = await db.all('SELECT * FROM suggestions');
+  for (const rows of res) {
+    console.log(rows['id'], rows['created_at'], rows['author'], rows['suggestion'], rows['state']);
+  }
+*/
+
+// All states of suggestion records
+enum SuggestionStates {
+  Create = 'Create',
+  Rejected = 'Rejected',
+  Accepted = 'Accepted',
+  Pending = 'Pending'
+}
+
+export const suggestCmd = async (message: Discord.Message, args: string[]): Promise<void> => {
+  const db = await openDB();
+
+  const helpArg = 'help';
+  let words = '';
+  // Turn args into suggestion
+  for (const word in args) {
+    words += args[word] + ' ';
+  }
+
+  if (words == '') {
+    message.channel.send('Codey sees an empty suggestion! Try again.');
+  } else if (args[0].toLowerCase() === helpArg) {
+    message.channel.send('.suggest <suggestion> \nYour <suggestion> should only contain text.');
```

Comment:
Can you throw the command example part into a code block? Like show it as `.suggest <suggestion>`

Reply:
We don't have to focus too much on formatting, I just want a little fidelity tis all